handlers/search.js

```diff
@@ -111,6 +112,39 @@ function getFilters(query) {
 
 function searchPage(request, reply) {
   const query = request.query;
+
+  const schema = Joi.object().keys({
+    page: Joi.number().integer().min(1),
+    registration_date_start: Joi.date().format('YYYY-MM-DD').empty(''),
+    registration_date_end: Joi.date().format('YYYY-MM-DD').empty(''),
+    location: Joi.array().single(true).items(Joi.string().empty('')),
+    q: Joi.string().empty(''),
+    sample_size_start: Joi.number().integer().empty(''),
+    sample_size_end: Joi.number().integer().empty(''),
+  });
+
+  const searchParams = {
+    page: query.page,
+    registration_date_start: query.registration_date_start,
+    registration_date_end: query.registration_date_end,
+    location: query.location,
+    q: query.q,
+    sample_size_start: query.sample_size_start,
+    sample_size_end: query.sample_size_end,
+  };
```

Comment:
Why are you using `searchParams` instead of `request.query`? Ideally, we would use `request.query`, so we catch unrecognized query parameters as well.